.github/workflows/docker.yml

```diff
@@ -0,0 +1,38 @@
+name: Docker CI
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        # Only push if on main branch
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          file: ./docker/alpine.Dockerfile
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: github.ref == 'refs/head/main' && github.event_name == 'push'
+          tags: marigold-dev/deku:latest
```

Comment:
We need commit hash in the tags to trace things when deployments need debugging.

Reply:
*too

Reply:
Not saying `latest` images are not needed. Perhaps another PR could deal with such tags

Reply:
I need to fix the boolean expression above, will add a commit hash tag as well 👌

Reply:
Will tag with `latest` and the git sha